localstack/services/s3/utils.py

```diff
@@ -70,15 +79,16 @@
 RFC1123 = "%a, %d %b %Y %H:%M:%S GMT"
 
 
-class InvalidRequest(ServiceException):
-    code: str = "InvalidRequest"
-    sender_fault: bool = False
-    status_code: int = 400
+def get_owner_for_account_id(account_id: str):
+    return Owner(
```

Comment:
I will, it's basically a hardcoded string, not exactly sure yet basically, it was hardcoded in moto, but I'd rather have a method so that we extend it later on.